app/controllers/article_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ArticleController < ApplicationController
+	before_action :authenticate_user!
+	def create
+		article = Article.new(article_params)
```

Comment:
this is not the right way.
Article.new(:title => article_params[:title], :body => article_params[:body] ) => pass with attributes